docs/articles/tutorial/1-welcome.md

```diff
@@ -0,0 +1,35 @@
+# Welcome
```

Comment:
Generally, I feel like this article is in some places too "nothing" (like it's not saying/explaining anything, yet has many words that may confuse the reader), while in other places it goes too specific (like init containers vs containers, where the User at this point have no clue what Testkube is)